ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -0,0 +1,172 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+open class ContactView: UIView {
+    private struct Constants {
+        static let contactWidth: CGFloat = 70.0
+        static let avatarViewWidth: CGFloat = 70.0
+        static let avatarViewHeight: CGFloat = 70.0
+        static let firstNameLabelMinimumHeight: CGFloat = 12.0
+        static let spacingBetweenAvatarAndFirstLabel: CGFloat = 13.0
+    }
+
+    private var avatarView: AvatarView
+    private var firstNameLabel: UILabel?
+    private var lastNameLabel: UILabel?
+    private var identifierLabel: UILabel?
+
+    /// Initializes the contact view with an avatar view, first name, and last name
+    ///
+    /// - Parameters:
+    ///   - avatarView: The AvatarView that will be displayed within the ContactView
+    ///   - firstName: String that will be the text of the top label
+    ///   - lastName: String that will be the text of the bottom label
+    public init(avatarView: AvatarView, firstName: String, lastName: String) {
+        self.avatarView = avatarView
+        self.firstNameLabel = UILabel()
+        self.lastNameLabel = UILabel()
+        super.init(frame: .zero)
+        self.backgroundColor = Colors.surfacePrimary
+        self.translatesAutoresizingMaskIntoConstraints = false
+        setupFirstNameLabel(using: firstName)
+        setupLastNameLabel(using: lastName)
+        setupLayout()
+    }
+
+    /// Initializes the contact view with an avatar view and an identifier
+    ///
+    /// - Parameters:
+    ///   - avatarView: The AvatarView that will be displayed within the ContactView
+    ///   - identifier: String that will be used to identify the contact (e.g. email, phone number, first name)
+    public init(avatarView: AvatarView, identifier: String) {
+        self.avatarView = avatarView
+        super.init(frame: .zero)
+        self.backgroundColor = Colors.surfacePrimary
+        self.translatesAutoresizingMaskIntoConstraints = false
+        setupIdentifierLabel(using: identifier)
+        setupLayout()
+    }
+
+    public required init?(coder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
+    private func setupLayout() {
+        var constraints = [NSLayoutConstraint]()
+        constraints.append(self.widthAnchor.constraint(equalToConstant: Constants.contactWidth))
+        constraints.append(contentsOf: avatarLayoutConstraints())
+
+        if firstNameLabel != nil && lastNameLabel != nil {
+            constraints.append(contentsOf: firstNameLayoutConstraints())
+            constraints.append(contentsOf: lastNameLayoutConstraints())
+        } else {
+            constraints.append(contentsOf: identifierLayoutConstraints())
+        }
+
+        NSLayoutConstraint.activate(constraints)
+    }
+
+    private func avatarLayoutConstraints() -> [NSLayoutConstraint] {
+        addSubview(avatarView)
```

Comment:
>addSubview [](start = 8, length = 10)

can we move this out of this method? The method name doesn't indicate that an action like this will take place